### ISSUE TITLE
Pi-Hole option

### ISSUE THREAD

**Issue by revilo951** [jlund/streisand-discussions#172]
Maybe straying a _little_ from the scope of the (awesome!) project, but maybe a customization option could be to have Streisand install a Pi-Hole for DNS-based ad-blocking for any connected users?

**Comment by Keridoo** [comment]
+1

**Comment by Frichetten** [comment]
This has actually come up a few times, #41, #170, #98, for example. I don't know if this will ever be a feature, but you can definitely setup Pihole on your Streisand instance after the fact. I do this on mine and it works great.

**Comment by christopher-owen** [comment]
> 
> 
> This has actually come up a few times, #41, #170, #98, for example. I don't know if this will ever be a feature, but you can definitely setup Pihole on your Streisand instance after the fact. I do this on mine and it works great.

Hey - I have Pi-hole installed on a Streisand instance and it works but I have to use cli to whitelist and view the query log etc. Have you got the Pi-hole admin web interface working too? Also I mainly use Wireguard on my phones which makes use of Pi-hole, but Shadowsocks, for example, doesn't. Have you got all the different Streisand protocols making use of Pi-hole? Cheers

**Comment by Frichetten** [comment]
@christopher-owen, Yeah, I have the web interface working okay. When I setup Pi-Hole I chose the IP address to set it on and this is the IP I reach the web interface on. For example, 192.168.122.1/admin. 

Unfortunately, I have not gotten all of the protocols to work :( I've mainly used OpenVPN and Wireguard. That being said, I think it would be possible to make all of them work, you'd just need to point them appropriately.

**Comment by TheNomad11** [comment]
I was not able to access admin pages by the URL. I made it work by using this guide https://docs.pi-hole.net/guides/nginx-configuration/ and remove the streisand nginx configuration that I anyway don't seem to need
